server/sonar-server/src/test/java/org/sonar/server/computation/task/projectanalysis/component/ConfigurationRepositoryTest.java

```diff
@@ -107,6 +106,19 @@ public void project_settings_are_cached_to_avoid_db_access() {
     assertThat(config.get("key")).hasValue("value");
   }
 
+  @Test
+  @Ignore
+  public void get_branch_settings_from_db() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)